### DISCUSSION TITLE
Request deletion/access to old/stagnant account

### DISCUSSION THREAD

**Discussion by matthewtownson** [community/community#34937] [Other Feature Feedback, Questions, & Ideas] [answered]
I am looking to create an organisation for my workplace for us to store our source code more centrally. There is an existing account with the name we would ideally like to use, however, the account has never had any activity and has no information. It just looks like a placeholder somebody set up, like https://github.com/Cken (this is not the name I am interested in, just a similar looking example). Is there a way to request this account is deleted so I can take over the name?

I realise that it may be active and just have activity hidden from me, so this may not be possible. The only options on the web interface are to report the account, which I don't think matches my request.

**Comment by airtower-luna** [comment, marked as answer]
GitHub staff have stated that they don't release usernames on request, here's a very detailed post: https://github.com/orgs/community/discussions/28516#discussioncomment-3337421

**Reply by matthewtownson** [reply]
I suspected but wanted to be sure. My google-fu was not good enough to find that, thanks for pointing me to it.
